README.md

```diff
@@ -158,14 +167,15 @@ cf. https://github.com/gree/unity-webview/issues/785
 cf. https://github.com/gree/unity-webview/issues/224#issuecomment-640642516
 
 ### Android
-
 Since `Assets/Plugins/src` is deprecated in recent Unity versions, we switched to .aar files (Android Archive Library).
 If you need to edit some AndroidManifest.xml files :
 1. Opening the `plugin`folder of this repo in Android Studio
 2. Make somes changes (most of the time edit the `plugin/Android/webview/src/main/AndroidManifest.xml`).
 3. Build the app
 4. You will see your aar file in `build/outputs/aar/`
 5. Move this file to `Assets/Plugins/Android/`
+#### How does it work ?
+It works on Android by calling a new android activity which implements a webview. We pass arguments from Unity to this activity to control it.   
```

Comment:
What does this part describe? The current implementation doesn't utilize any special activity.